### PR TITLE
Fix sv curation and merge units with array properties

### PR DESCRIPTION
**Pull request by alejoe91**

### CODE REVIEW

File: src/spikeinterface/curation/mergeunitssorting.py [hunk 1]
```diff
@@ -105,12 +105,20 @@ def __init__(self, parent_sorting, units_to_merge, new_unit_ids=None, properties
 
             if properties_policy == "keep":
                 # propagate keep values
-                new_values = np.empty(shape=len(units_ids), dtype=parent_values.dtype)
+                if isinstance(parent_values, np.ndarray):
```
Comment (alejoe91) [inline src/spikeinterface/curation/mergeunitssorting.py:108]: This is needed to handle array properties

File: src/spikeinterface/widgets/sorting_summary.py [hunk 2]
```diff
@@ -35,9 +35,6 @@ class SortingSummaryWidget(BaseWidget):
     curation : bool, default: False
         If True, manual curation is enabled
         (sortingview backend)
-    unit_table_properties : list or None, default: None
```
Comment (alejoe91) [inline src/spikeinterface/widgets/sorting_summary.py:38]: duplicated

File: src/spikeinterface/curation/mergeunitssorting.py [hunk 3]
```diff
@@ -105,12 +105,20 @@ def __init__(self, parent_sorting, units_to_merge, new_unit_ids=None, properties
...
+                    shape = (len(units_ids),) + parent_values.shape[1:]
```
Comment (alejoe91) [inline src/spikeinterface/curation/mergeunitssorting.py:109]: get rid of if else, this will always work

**Comment by alejoe91** [comment]
@samuelgarcia done! cleaned up and added some tests too

**Review by samuelgarcia — APPROVED** [review]

**Pull request merged**